src/collections/_documentation/platforms/native/index.md

```diff
@@ -398,42 +379,14 @@ The transport is invoked in the same thread as the call to
 `sentry_capture_event`. Consider to offload network communication to a
 background thread or thread pool to avoid blocking execution.
 
-## Integrations
-
-*Integrations* extend the functionality of the SDK for some common frameworks
-and libraries. Similar to plugins, they extend the functionality of the Sentry
-SDK. The Native SDK comes in two additional distributions that include
-integrations into the Breakpad and Crashpad libraries, respectively.
-
-### Google Breakpad
+## Backends
```

Comment:
Let's name this `Integrations` still. In public docs, we always talk about integrations for these kinds of thiings.